in_toto/runlib.py

```diff
@@ -82,6 +92,14 @@ def record_artifacts_as_dict(artifacts):
     The files that result form a link command execution are called
     products.
 
+    Excludes files that are matched by the file patterns specified in
+    ARTIFACT_EXCLUDES setting.
+    Note: Exclude patterns are applied on the basename, i.e. the pattern
+    "foo" excludes /foo as well as /subdir/foo
```

Comment:
Hmm, I'm assuming it would be a good idea to mimic the behavior/semantic that .gitignore does to simplify adoption. Both files, gitignore and in-totoignore, serve the same purpose, so why not?

We could ticketize it and move forward with the PR though.